MobiFlight/MidiBoard/MidiBoardManager.cs

```diff
@@ -80,24 +78,27 @@ private List<string> CollectAndLogDefinitionErrors(MidiBoardDefinition def)
         }
 
 
-        private void LoadDefinitions()
+        private void Load()
```

Comment:
Old name was more clearly telling what is loaded here

Reply:
Renamed back